backend/danswer/background/indexing/run_indexing.py

```diff
@@ -382,46 +381,15 @@ def _run_indexing(
 def _prepare_index_attempt(
     db_session: Session, index_attempt_id: int, tenant_id: str | None
```

Comment:
Believe that tenant id argument and db session execution here are unnecessary (entry point seems to set proper schema)